### PR TITLE
Update release notes for 19.8

### PR DESCRIPTION
**Pull request by pmusolino**
This PR updates the 19.8 release notes and changelog files as part of the code freeze process.

---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.
- [ ] This PR includes refactoring; smoke testing of the entire section is needed.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR has the <code>Releases</code> label: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <!--
 Contains editorialized release notes. Raw release notes should go into `RELEASE-NOTES.txt`.
 -->
+## 19.8
+Experience seamless updates with our latest release! Now enjoy background updates for your order list and dashboard analytics cards. We've enhanced the dynamic dashboard to display unavailable analytics views when not logged into WPCom. Plus, we've fixed large text support in the Bar Code Scanner for better accessibility. Update now for a smoother experience!
```
Comment (toupper) [inline CHANGELOG.md]: Bar code feels more natural as a single word to me: Barcode Scanner

**Review by toupper — APPROVED** [review]
Just a small remark :)

File: CHANGELOG.md (hunk 1 above)
Comment (pmusolino) [reply CHANGELOG.md]: Thanks, for the suggestion! a5ac612d6d7ae923a9ff7e5640744c945b980d60

**Pull request merged**